### PR TITLE
[doc] Add doc for linux_script and windows_script

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
## What's changed?

close https://github.com/apache/hertzbeat/issues/2268
Add doc for linux_script and windows_script

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [x]  I have written the necessary doc or comment.

### CODE REVIEW

File: home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/linux_script.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+### 准备工作
+
+> 若要监控本机，部署 Hertzbeat 即可，若要监控其他主机，需在目标主机上部署采集器，部署采集器可参考[此链接](https://github.com/apache/hertzbeat?tab=readme-ov-file#2install-via-package)的第 5 步。
+> 在创建监控任务选择采集器时，需选择目标主机上对应的采集器。
+
+### 配置参数
+
+| 参数名称   |             参数帮助描述              |
+|:-------|---------------------------------|---|
+| 监控Host | 被监控的对端IPV4，IPV6或域名。注意⚠️不带协议头(eg: https://, http://)。           |
```
Comment (yuluo-yx) [inline home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/linux_script.md:17]: add some blanks. such as `监控 Host，被监控的对端 IPV4...`

File: home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/linux_script.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+|----------|------|--------|
+| hostname | 无    | 主机名称   |
+| version  | 无    | 操作系统版本 |
+| uptime   | 无    | 系统启动时间 |
+
+#### 指标集合：cpu
+
+| 指标名称           | 指标单位 | 指标帮助描述             |
+|----------------|------|--------------------|
+| info           | 无    | CPU型号              |
```
Comment (yuluo-yx) [inline home/i18n/zh-cn/docusaurus-plugin-content-docs/current/help/linux_script.md:38]: ditto

**Comment by yuluo-yx** [comment]
hi, you can run `cd home && yarn md-lint` to check md style.

**Comment by zuobiao-zhou** [comment]
> hi, you can run `cd home && yarn md-lint` to check md style.

ok, thanks!

**Comment by yuluo-yx** [comment]
> > hi, you can run `cd home && yarn md-lint` to check md style.
> 
> ok, thanks!

more information you can see: https://github.com/apache/hertzbeat/pull/2640/files#diff-14797f4558afc57d64d406d4d19b829d3a0ec594fcc19cf846398cde4c266dba

**Review by yuluo-yx — APPROVED** [review]
LGTM

**Pull request merged**